x-pack/plugins/apm/public/components/shared/managed_table/index.tsx

```diff
@@ -41,16 +45,25 @@ interface Props<T extends object> {
   isLoading?: boolean;
   error?: boolean;
   tableLayout?: 'auto' | 'fixed';
+  tableSearchBar?: {
+    isEnabled?: boolean;
+    fieldsToSearch: string[];
+    maxCountExceeded: boolean;
+    isSearchSideSearchQueryActive: boolean;
```

Comment:
Oh, that's a typo. Should be `isServerSideSearchQueryActive`. Still quiet wordy but definitely better

Reply:
... actually, we might be able to simplify this a bunch by just removing this. It will cause the server to be requested a bit unnecessarily but might not be a big issue. One case to highlight: if all results have been fetched from the server, we do not need to request it again. However, if the user presses "backspace" to delete a character, we cannot know for sure if the server has more results, without `isServerSideSearchQueryActive`.